.github/workflows/build_and_push.yml

```
@@ -98,7 +98,7 @@ jobs:
         with:
           file: ${{ matrix.services.dockerfile }}
           context: ${{ matrix.services.docker_context }}
-          push: ${{ matrix.services.enable_dockerhub && (github.event_name == 'release' || github.ref_name == 'master') }}
+          push: ${{ matrix.services.enable_dockerhub && (github.event_name == 'tag' || github.ref_name == 'master') }}
```

```suggestion
          push: ${{ matrix.services.enable_dockerhub && github.event_name == 'tag' }}
```

Am wondering if we need all these `commit-sha` tagged images pushed, when a push happens on master branch ? What do you think ?